Dockerfile.functional

```diff
@@ -36,6 +36,10 @@ RUN yum install -y wget \
     && rm -f google-chrome-stable_current_*.rpm \
     && yum clean all
 
+ENV OCP_VERSION=stable-4.19
```

Comment:
Not sure I like adding the openshift version here. Leaves a dependency that will need to be maintained.  How about we use the latest instead https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz